src/Asm89/Stack/CorsService.php

```diff
@@ -36,16 +36,13 @@ private function normalizeOptions(array $options = array())
         );
 
         // normalize array('*') to true
-        if (in_array('*', $options['allowedOrigins'])) {
+        if ($options['allowedOrigins'] === true || in_array('*', $options['allowedOrigins'])) {
```

Comment:
This is to ease configuration? `$options['allowedOrigins'] = true` does read a bit odd to me.